### PR TITLE
Implement Action Update  

### PR DESCRIPTION
**Pull request by kl52752**
* Fix for generic action update
* support for action update in health check
* support for fingerprint
* support for action update in backend service

### CODE REVIEW

**Comment by kl52752** [comment]
assign/ @bowei

**Comment by kl52752** [comment]
cc/ @mag-kol 
cc/ @awesomepatrol

**Review by bowei — COMMENTED** [review]
There is a _major_ todo in the BackendService implementation, but let's get this in so people are unblocked.

/lgtm
/approve

File: pkg/cloud/rgraph/rnode/backendservice/node.go [hunk 1]
```diff
@@ -81,12 +86,38 @@ func (n *backendServiceNode) Diff(gotNode rnode.Node) (*rnode.PlanDetails, error
...
+			return "", err
+		}
+		return obj.Fingerprint, nil
+	case meta.VersionAlpha:
+		obj, err := gotRes.ToAlpha()
+		if err != nil {
+			return "", err
+		}
+		return obj.Fingerprint, nil
+
```
Comment (AwesomePatrol) [inline pkg/cloud/rgraph/rnode/backendservice/node.go:110]: nit: style consistency
```suggestion

```

**Pull request merged**